uberAgentSupport-macOS/uberAgentSupport.zsh

```diff
@@ -40,27 +40,56 @@ if [ "$(id -u)" != "0" ]; then
 fi
 
 # Create a structured directory tree inside the bundle path
-mkdir -p "${bundlePath}"/{Config/Local,Config/CCFM,Logs/Daemon,Logs/Users,CrashReports}
+mkdir -p "${bundlePath}"/{Config/Local,Config/CCFM,Logs/Daemon,Logs/Users,Logs/Remote,CrashReports}
```

Comment:
Optional: add checks before this line to ensure hostname and bundlePath are not empty.

```
if [ -z "${bundlePath}" ]; then
    echo "Error: bundlePath is not set."
    exit 1
fi

if [ -z "${hostname}" ]; then
    echo "Error: hostname is not set."
    exit 1
fi
```

Reply:
Added check for bundlePath.